### PR TITLE
Added support for lemonbar

### PR DESCRIPTION
**Pull request by bsvh**
This adds support for lemonbar (https://github.com/LemonBoy/bar).

### CODE REVIEW

File: i3status.c [hunk 1]
```diff
@@ -163,11 +163,18 @@ static int parse_min_width(cfg_t *context, cfg_opt_t *option, const char *value,
...
-        return 0;
+    int len;
+
+    /* lemonbar supports an optional alpha channel */
+    if ((len = strlen(value)) != 7) {
+        if (output_format != O_LEMONBAR)
+            return 0;
+        else if (len != 9)
+            return 0;
+    }
```
Comment (stapelberg) [inline i3status.c]: I think the following code makes it a bit easier to understand the logic:
```c
const int len = strlen(value);
if (output_format == O_LEMONBAR) {
  /* lemonbar supports an optional alpha channel */
  if (len != strlen("#rrggbb") && len != strlen("#rrggbbaa")) {
    return 0;
  }
} else {
  if (len != strlen("#rrggbb")) {
    return 0;
  }
}
```

**Comment by stapelberg** [comment]
Looks good overall.

**Comment by bsvh** [comment]
Ok, I updated the color validation like you asked.

**Comment by stapelberg** [comment]
Can you please use clang-format to format your code?

**Comment by Airblader** [comment]
And maybe also squash those commits. ;)

**Comment by bsvh** [comment]
Formatted and squashed.

**Pull request merged**